mne/report.py

```diff
@@ -259,6 +263,12 @@ def _update_html(html, report_fname, report_sectionlabel):
                 report_fname = fname
                 report_sectionlabel = 'raw'
             elif fname.endswith(('-fwd.fif', '-fwd.fif.gz')):
+                if report.subjects_dir is not None:
+                    html = report._render_sensitivity_map(fname, report.subjects_dir)
+                    report_fname = fname + '(sensitivity)'
```

Comment:
In case of a `*fwd.fif` type input, and also if `subjects_dir` is not None(which is required for sensitivity map plots), an extra html section is inserted in report with it's corresponding `sectionlabel` in navbar. And fname is appended with `(sensitivity)` for giving a different filename, to basically differentiate this section for a separate entry in Table of contents.